tasks/discover-granules/index.js

```diff
@@ -288,12 +289,7 @@ const discoverGranules = async ({ config }) => {
 
   const discoveredGranules = map(filesByGranuleId, buildGranule(config));
 
-  // Set the environment variable for the logger
-  if (discoveredGranules) {
-    process.env.GRANULES = JSON.stringify(discoveredGranules.map((g) => g.granuleId));
-  }
-
-  logger().info(`Discovered ${discoveredGranules.length} granules.`);
+  logger({ granules: discoveredGranules.map((g) => g.granuleId) }).info(`Discovered ${discoveredGranules.length} granules.`);
```

Comment:
Although the logger itself does a JSON.stringify so I'm not sure this would improve the formatting.

Reply:
So the typescript expects a string, but it looks much better if you provide it an array in cloudwatch.

```
logger({ granules: discoveredGranules.map((g) => g.granuleId) }).info(`Discovered ${discoveredGranules.length} granules.`);
```
```
{"granules":["granule-0","granule-1","granule-2","granule-3","granule-4","granule-5","granule-6","granule-7","granule-8","granule-9","granule-10","granule-11","granule-12","granule-13","granule-14"],"level":"info","message":"Discovered 15 granules.","sender":"@cumulus/discover-granules","timestamp":"2021-03-31T14:57:02.587Z"}
```
vs 
 
```
logger({ granules: JSON.stringify(discoveredGranules.map((g) => g.granuleId)) }).info(`Discovered ${discoveredGranules.length} granules.`);
```
```
{"granules":"[\"granule-0\",\"granule-1\",\"granule-2\",\"granule-3\",\"granule-4\",\"granule-5\",\"granule-6\",\"granule-7\",\"granule-8\",\"granule-9\",\"granule-10\",\"granule-11\",\"granule-12\",\"granule-13\",\"granule-14\"]","level":"info","message":"Discovered 15 granules.","sender":"@cumulus/discover-granules","timestamp":"2021-03-31T14:57:28.332Z"}
```
Additionally cloudwatch will put each granule on a separate line up until there are too many characters.  All of my debugging logs are gone because I have redeployed.

So, long story.  You can say "use a string because the TS expects it", or trust me that it looks better as an array.

Reply:
I think this is what you meant. 8aba223691b95b8cdc084c1a9d3c0335449f2cb0